css/scss/_page-home.scss

```diff
@@ -0,0 +1,72 @@
+.sfh-wrapper {
+  margin: auto 0;
+  padding: 50px 50px;
+}
+
+.sfh-title {
+  @include primaryFontStack;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: $white;
+  letter-spacing: 0.1em;
+}
+
+.sfh-name {
+  @include primaryFontStack;
+  font-size: 36px;
+  color: $white;
+}
+
+.sfh-copy {
+  @include primaryFontStackLight;
+  font-size: 18px;
+  color:  $white;
+}
+
+.exercise-buttons {
+  padding: 25px 50px;
+  text-align: center;
+
+  @supports (grid-area: auto) {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-gap: 10px;
+  }
+
+  @include respond-screen ($bp-md) {
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 30px;
+  }
+}
+
+.button-one {
+  background: $button-yellow; 
+}
```

Comment:
@praxoc , please add an empty line between blocks. Some of them have but some of them don't.